src/Guard/ProductVariantInWishlistGuard.php

```diff
@@ -17,10 +17,12 @@
 
 final class ProductVariantInWishlistGuard implements ProductVariantInWishlistGuardInterface
 {
-    public function check(WishlistInterface $wishlist, ProductVariantInterface $productVariant): void
+    public function check(WishlistInterface $wishlist, ProductVariantInterface $productVariant): bool
```

Comment:
To be honest I don't see any reason of having this class, as we can use the method from WishlistInterface

Reply:
Done.